Synthesis failed for nodejs-logging

Hello! Autosynth couldn't regenerate nodejs-logging. :broken_heart:

Here's the output from running `synth.py`:

```
S2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'LogView'.

[7m2095[0m                 stream.on('data', (response: protos.google.logging.v2.LogView) => {
[7m    [0m [91m                                                                      ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2122[0m:[93m80[0m - [91merror[0m[90m TS2551: [0mProperty 'ListViewsRequest' does not exist on type 'typeof v2'. Did you mean 'ListSinksRequest'?

[7m2122[0m             const request = generateSampleMessage(new protos.google.logging.v2.ListViewsRequest());
[7m    [0m [91m                                                                               ~~~~~~~~~~~~~~~~[0m

  [96mprotos/protos.d.ts[0m:[93m2661[0m:[93m19[0m
    [7m2661[0m             class ListSinksRequest implements IListSinksRequest {
    [7m    [0m [96m                  ~~~~~~~~~~~~~~~~[0m
    'ListSinksRequest' is declared here.

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2129[0m:[93m59[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'LogView'.

[7m2129[0m                 const responses: protos.google.logging.v2.LogView[] = [];
[7m    [0m [91m                                                          ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2130[0m:[93m71[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'LogView'.

[7m2130[0m                 stream.on('data', (response: protos.google.logging.v2.LogView) => {
[7m    [0m [91m                                                                      ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2156[0m:[93m80[0m - [91merror[0m[90m TS2551: [0mProperty 'ListViewsRequest' does not exist on type 'typeof v2'. Did you mean 'ListSinksRequest'?

[7m2156[0m             const request = generateSampleMessage(new protos.google.logging.v2.ListViewsRequest());
[7m    [0m [91m                                                                               ~~~~~~~~~~~~~~~~[0m

  [96mprotos/protos.d.ts[0m:[93m2661[0m:[93m19[0m
    [7m2661[0m             class ListSinksRequest implements IListSinksRequest {
    [7m    [0m [96m                  ~~~~~~~~~~~~~~~~[0m
    'ListSinksRequest' is declared here.

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2159[0m:[93m66[0m - [91merror[0m[90m TS2339: [0mProperty 'LogView' does not exist on type 'typeof v2'.

[7m2159[0m               generateSampleMessage(new protos.google.logging.v2.LogView()),
[7m    [0m [91m                                                                 ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2160[0m:[93m66[0m - [91merror[0m[90m TS2339: [0mProperty 'LogView' does not exist on type 'typeof v2'.

[7m2160[0m               generateSampleMessage(new protos.google.logging.v2.LogView()),
[7m    [0m [91m                                                                 ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2161[0m:[93m66[0m - [91merror[0m[90m TS2339: [0mProperty 'LogView' does not exist on type 'typeof v2'.

[7m2161[0m               generateSampleMessage(new protos.google.logging.v2.LogView()),
[7m    [0m [91m                                                                 ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2164[0m:[93m55[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'ILogView'.

[7m2164[0m             const responses: protos.google.logging.v2.ILogView[] = [];
[7m    [0m [91m                                                      ~~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2186[0m:[93m80[0m - [91merror[0m[90m TS2551: [0mProperty 'ListViewsRequest' does not exist on type 'typeof v2'. Did you mean 'ListSinksRequest'?

[7m2186[0m             const request = generateSampleMessage(new protos.google.logging.v2.ListViewsRequest());
[7m    [0m [91m                                                                               ~~~~~~~~~~~~~~~~[0m

  [96mprotos/protos.d.ts[0m:[93m2661[0m:[93m19[0m
    [7m2661[0m             class ListSinksRequest implements IListSinksRequest {
    [7m    [0m [96m                  ~~~~~~~~~~~~~~~~[0m
    'ListSinksRequest' is declared here.

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2192[0m:[93m59[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'ILogView'.

[7m2192[0m                 const responses: protos.google.logging.v2.ILogView[] = [];
[7m    [0m [91m                                                          ~~~~~~~~[0m


Found 170 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging@8.1.0 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging@8.1.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-11-14T12_26_14_750Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging@8.1.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging@8.1.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-11-14T12_26_14_804Z-debug.log
2020-11-14 04:26:14,837 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging/synth.py", line 40, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-11-14 04:26:14,907 autosynth [ERROR] > Synthesis failed
2020-11-14 04:26:14,907 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b4e3f89 docs(sample): improve advanced log sample (#933)
2020-11-14 04:26:14,919 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-14 04:26:14,928 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0512ff9f-78b8-4548-bf36-503f3504dfab/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
S2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'LogView'.

[7m2095[0m                 stream.on('data', (response: protos.google.logging.v2.LogView) => {
[7m    [0m [91m                                                                      ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2122[0m:[93m80[0m - [91merror[0m[90m TS2551: [0mProperty 'ListViewsRequest' does not exist on type 'typeof v2'. Did you mean 'ListSinksRequest'?

[7m2122[0m             const request = generateSampleMessage(new protos.google.logging.v2.ListViewsRequest());
[7m    [0m [91m                                                                               ~~~~~~~~~~~~~~~~[0m

  [96mprotos/protos.d.ts[0m:[93m2661[0m:[93m19[0m
    [7m2661[0m             class ListSinksRequest implements IListSinksRequest {
    [7m    [0m [96m                  ~~~~~~~~~~~~~~~~[0m
    'ListSinksRequest' is declared here.

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2129[0m:[93m59[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'LogView'.

[7m2129[0m                 const responses: protos.google.logging.v2.LogView[] = [];
[7m    [0m [91m                                                          ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2130[0m:[93m71[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'LogView'.

[7m2130[0m                 stream.on('data', (response: protos.google.logging.v2.LogView) => {
[7m    [0m [91m                                                                      ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2156[0m:[93m80[0m - [91merror[0m[90m TS2551: [0mProperty 'ListViewsRequest' does not exist on type 'typeof v2'. Did you mean 'ListSinksRequest'?

[7m2156[0m             const request = generateSampleMessage(new protos.google.logging.v2.ListViewsRequest());
[7m    [0m [91m                                                                               ~~~~~~~~~~~~~~~~[0m

  [96mprotos/protos.d.ts[0m:[93m2661[0m:[93m19[0m
    [7m2661[0m             class ListSinksRequest implements IListSinksRequest {
    [7m    [0m [96m                  ~~~~~~~~~~~~~~~~[0m
    'ListSinksRequest' is declared here.

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2159[0m:[93m66[0m - [91merror[0m[90m TS2339: [0mProperty 'LogView' does not exist on type 'typeof v2'.

[7m2159[0m               generateSampleMessage(new protos.google.logging.v2.LogView()),
[7m    [0m [91m                                                                 ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2160[0m:[93m66[0m - [91merror[0m[90m TS2339: [0mProperty 'LogView' does not exist on type 'typeof v2'.

[7m2160[0m               generateSampleMessage(new protos.google.logging.v2.LogView()),
[7m    [0m [91m                                                                 ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2161[0m:[93m66[0m - [91merror[0m[90m TS2339: [0mProperty 'LogView' does not exist on type 'typeof v2'.

[7m2161[0m               generateSampleMessage(new protos.google.logging.v2.LogView()),
[7m    [0m [91m                                                                 ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2164[0m:[93m55[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'ILogView'.

[7m2164[0m             const responses: protos.google.logging.v2.ILogView[] = [];
[7m    [0m [91m                                                      ~~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2186[0m:[93m80[0m - [91merror[0m[90m TS2551: [0mProperty 'ListViewsRequest' does not exist on type 'typeof v2'. Did you mean 'ListSinksRequest'?

[7m2186[0m             const request = generateSampleMessage(new protos.google.logging.v2.ListViewsRequest());
[7m    [0m [91m                                                                               ~~~~~~~~~~~~~~~~[0m

  [96mprotos/protos.d.ts[0m:[93m2661[0m:[93m19[0m
    [7m2661[0m             class ListSinksRequest implements IListSinksRequest {
    [7m    [0m [96m                  ~~~~~~~~~~~~~~~~[0m
    'ListSinksRequest' is declared here.

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2192[0m:[93m59[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'ILogView'.

[7m2192[0m                 const responses: protos.google.logging.v2.ILogView[] = [];
[7m    [0m [91m                                                          ~~~~~~~~[0m


Found 170 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging@8.1.0 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging@8.1.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-11-15T12_26_56_458Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging@8.1.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging@8.1.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-11-15T12_26_56_525Z-debug.log
2020-11-15 04:26:56,555 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging/synth.py", line 40, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-11-15 04:26:56,628 autosynth [ERROR] > Synthesis failed
2020-11-15 04:26:56,628 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b4e3f89 docs(sample): improve advanced log sample (#933)
2020-11-15 04:26:56,640 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-15 04:26:56,650 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/dad05890-a786-4d1a-96ef-748c78703334/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Here's the output from running `synth.py`:

```
S2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'LogView'.

[7m2095[0m                 stream.on('data', (response: protos.google.logging.v2.LogView) => {
[7m    [0m [91m                                                                      ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2122[0m:[93m80[0m - [91merror[0m[90m TS2551: [0mProperty 'ListViewsRequest' does not exist on type 'typeof v2'. Did you mean 'ListSinksRequest'?

[7m2122[0m             const request = generateSampleMessage(new protos.google.logging.v2.ListViewsRequest());
[7m    [0m [91m                                                                               ~~~~~~~~~~~~~~~~[0m

  [96mprotos/protos.d.ts[0m:[93m2661[0m:[93m19[0m
    [7m2661[0m             class ListSinksRequest implements IListSinksRequest {
    [7m    [0m [96m                  ~~~~~~~~~~~~~~~~[0m
    'ListSinksRequest' is declared here.

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2129[0m:[93m59[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'LogView'.

[7m2129[0m                 const responses: protos.google.logging.v2.LogView[] = [];
[7m    [0m [91m                                                          ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2130[0m:[93m71[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'LogView'.

[7m2130[0m                 stream.on('data', (response: protos.google.logging.v2.LogView) => {
[7m    [0m [91m                                                                      ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2156[0m:[93m80[0m - [91merror[0m[90m TS2551: [0mProperty 'ListViewsRequest' does not exist on type 'typeof v2'. Did you mean 'ListSinksRequest'?

[7m2156[0m             const request = generateSampleMessage(new protos.google.logging.v2.ListViewsRequest());
[7m    [0m [91m                                                                               ~~~~~~~~~~~~~~~~[0m

  [96mprotos/protos.d.ts[0m:[93m2661[0m:[93m19[0m
    [7m2661[0m             class ListSinksRequest implements IListSinksRequest {
    [7m    [0m [96m                  ~~~~~~~~~~~~~~~~[0m
    'ListSinksRequest' is declared here.

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2159[0m:[93m66[0m - [91merror[0m[90m TS2339: [0mProperty 'LogView' does not exist on type 'typeof v2'.

[7m2159[0m               generateSampleMessage(new protos.google.logging.v2.LogView()),
[7m    [0m [91m                                                                 ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2160[0m:[93m66[0m - [91merror[0m[90m TS2339: [0mProperty 'LogView' does not exist on type 'typeof v2'.

[7m2160[0m               generateSampleMessage(new protos.google.logging.v2.LogView()),
[7m    [0m [91m                                                                 ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2161[0m:[93m66[0m - [91merror[0m[90m TS2339: [0mProperty 'LogView' does not exist on type 'typeof v2'.

[7m2161[0m               generateSampleMessage(new protos.google.logging.v2.LogView()),
[7m    [0m [91m                                                                 ~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2164[0m:[93m55[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'ILogView'.

[7m2164[0m             const responses: protos.google.logging.v2.ILogView[] = [];
[7m    [0m [91m                                                      ~~~~~~~~[0m

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2186[0m:[93m80[0m - [91merror[0m[90m TS2551: [0mProperty 'ListViewsRequest' does not exist on type 'typeof v2'. Did you mean 'ListSinksRequest'?

[7m2186[0m             const request = generateSampleMessage(new protos.google.logging.v2.ListViewsRequest());
[7m    [0m [91m                                                                               ~~~~~~~~~~~~~~~~[0m

  [96mprotos/protos.d.ts[0m:[93m2661[0m:[93m19[0m
    [7m2661[0m             class ListSinksRequest implements IListSinksRequest {
    [7m    [0m [96m                  ~~~~~~~~~~~~~~~~[0m
    'ListSinksRequest' is declared here.

[96mtest/gapic_config_service_v2_v2.ts[0m:[93m2192[0m:[93m59[0m - [91merror[0m[90m TS2694: [0mNamespace '"/home/kbuilder/.cache/synthtool/nodejs-logging/protos/protos".google.logging.v2' has no exported member 'ILogView'.

[7m2192[0m                 const responses: protos.google.logging.v2.ILogView[] = [];
[7m    [0m [91m                                                          ~~~~~~~~[0m


Found 170 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging@8.1.0 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging@8.1.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-11-16T12_26_51_215Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging@8.1.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging@8.1.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-11-16T12_26_51_272Z-debug.log
2020-11-16 04:26:51,304 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging/synth.py", line 40, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-11-16 04:26:51,373 autosynth [ERROR] > Synthesis failed
2020-11-16 04:26:51,373 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b4e3f89 docs(sample): improve advanced log sample (#933)
2020-11-16 04:26:51,385 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-11-16 04:26:51,394 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/675f9f30-41fd-4452-ad1c-500c24bb8ba1/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "fix": "gts fix",
     "prelint": "cd samples; npm link ../; npm install",
     "lint": "gts check",
-    "prepare": "npm run compile",
+    "prepare": "npm run compile-protos && npm run compile",
     "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
     "presystem-test": "npm run compile",
     "system-test": "mocha build/system-test --timeout 600000",

